doc/howto/override-default-entrypoint.rst

```diff
@@ -0,0 +1,51 @@
+How to override the default entry point
```

Comment:
```suggestion
How to override the default OCaml entrypoint with C Stubs
```
My suggestion is not particularly good, but I think it is important to emphasise the link with stubs and with OCaml.

Reply:
Done.